### PR TITLE
Docs: Amend user guide

### PR DESCRIPTION
**Pull request by wrewsama**

### CODE REVIEW

**Review by Tex-Tang — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 1]
```diff
@@ -41,7 +58,7 @@ GithubContact is an address book **integrated with Github's Public API**, target
 **:information_source: Notes about the command format:**<br>
```
Comment (Tex-Tang) [inline docs/UserGuide.md:58]: Could you add a note to mention about prefix alias here? For example, running `add name/NAME` or `add n/NAME`  both works. Probably a table. You can refer to [here](https://github.com/AY2223S1-CS2103T-W08-2/tp/blob/master/src/main/java/seedu/address/logic/parser/CliSyntax.java)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -71,88 +88,184 @@ Example:
...
+After the command is executed:
+![after deleting](images/ug/delete_after.png)
+
+#### Find a Person : `find`
+
+Displays the persons that match a given keyword. After which, the filtered list can be reset using the `reset` command.
+
+Format: 
+- To display the matching persons: `find KEYWORD`
+- To return to the main list: `reset`
```
[the remark this answers is not in the document]
Comment (wrewsama) [reply docs/UserGuide.md]: Amended by separating the reset command into its own section.

File: docs/UserGuide.md (hunk 1 above)
Comment (wrewsama) [reply docs/UserGuide.md:60]: Added note on prefix aliases as well as a table listing all aliases for each attribute.

**Review by Tex-Tang — APPROVED** [review]

**Review by Tex-Tang — APPROVED** [review]
Well done @wrewsama !! I can tell you had put a lot of efforts into it. The user guides now are much more elaborative and cleaner.

**Review by arnav-ag — APPROVED** [review]
Thanks a lot for helping with the user guide! LGTM

**Pull request merged**